pkg/limayaml/defaults.go

```diff
@@ -106,7 +106,11 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 	}
 	for arch := range cpuType {
 		if IsNativeArch(arch) && IsAccelOS() {
-			cpuType[arch] = "host"
+			if HasHostCPU() {
+				cpuType[arch] = "host"
+			} else {
+				cpuType[arch] = "max"
```

Comment:
Probably better to leave empty (QEMU default)

Reply:
You mean remove `-cpu` argument altogether ?

Reply:
Yes